PWGHF/TableProducer/treeCreatorBplusToD0Pi.cxx

```diff
@@ -34,6 +34,8 @@ namespace o2::aod
 {
 namespace full
 {
+DECLARE_SOA_INDEX_COLUMN_FULL(Candidate, candidate, int, HfCandBplus, "_0");
+DECLARE_SOA_INDEX_COLUMN_FULL(HfCandBplusParticle, hfcandbplusparticle, int, McParticles, "_0");
```

Comment:
Why are these index columns? Are these tables supposed to be processed within O2? If not, I don't see a reason to use this column type.